api/src/handlers/helpers/structureInputData/affirmation.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  CommonStructuredDataInput,
+  getCommonStructuredDataInput,
+  YesNoMap,
+} from "./common";
+import { InputFields } from "../getAllInputsFromForm";
+
+export interface AffirmationStructuredDataInput
+  extends CommonStructuredDataInput {
+  affirmationDetails: InputFields;
+}
+
+export function getAffirmationStructuredDataInput(
+  formData: InputFields
+): AffirmationStructuredDataInput {
+  const {
+    country,
+    hasUkPassport,
+    ceremonyType,
+    partnerIsOppositeSex,
+    partnerIsLocal,
+    partnersAreOfAge,
+    oathType,
+    paymentMethod,
+  } = formData;
+
+  const nationalLabel = `Partner is a ${country} national`;
+
+  return {
+    affirmationDetails: {
+      "Has a UK passport": YesNoMap[hasUkPassport.toString()],
+      "Ceremony type": ceremonyType,
```

Comment:
I assume we can pull this from the component title until we know otherwise. we don't have to remap things in the same way as ORU